src/clj/witan/send/schemas.clj

```diff
@@ -103,8 +103,8 @@
                            [:max-academic-year AcademicYear]
                            [:needs s/Str]]))
 
-(def PopulationByAcademicYear
-  [{AcademicYear s/Int}])
+(def PopulationByCalendarAndAcademicYear
+  {CalendarYear {AcademicYear N}})
```

Comment:
I think in a future version we'll want to move all of this over to clojure.spec.